src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const sum = days * 40;
+
+  if (days >= 3 && days < 7) {
+    return sum - 20;
```

Comment:
The value '3' and '20' are literals and should be replaced with constants that describe what these values represent, such as 'MIN_DAYS_FOR_DISCOUNT' and 'SHORT_TERM_DISCOUNT'.